### ISSUE TITLE
Add manual refresh button for attribute OCR

### ISSUE THREAD

**Issue by brollb** [symbench/scm#428]

**Comment by umesh-timalsina** [comment]
Where should the button live?

**Comment by umesh-timalsina** [comment]
Here's a proposed set of changes and a discussion is necessary around these changes:
1. Change the input field used on the `ComponentLabelWidget.svelte` to use `@smui/text-filed`. The reason is because we already have smui integration and the [text fields](https://sveltematerialui.com/demo/textfield/) already achieve what we want. They have custom icons helper texts and much more.
2. Where should the knowledge about this refresh live? Should we change the ontology for minor changes like this, if not it needs to be hardcoded. I was thinking we could have an input field like the following.
![image](https://user-images.githubusercontent.com/11476842/149389060-1962e347-def7-4a59-9461-8a337dfe7092.png)
3. The material UI theme scss files now need to live on the top level not on the `Components/TitleBar`.

**Comment by brollb** [comment]
I think it looks good. I don't think we need any additional info in the ontology since it is really only needed for the annotations that already use OCR for their fields.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by umesh-timalsina** [comment]
I agree, but there are cases where a fields can have value and unit. Adding a refresh button just to the `value` should work in that case but is it the most intuitive?

**Comment by brollb** [comment]
Good point. We can't really refresh just one of the fields... We could add a refresh button to both. On refresh, we could run OCR then assign the text to the different fields (based on regex from the ontology). Whichever button was refreshed could simply set the field accordingly and the other field (ie, value/unit) could be discarded.

It might be nice to first implement the first part of that so we can enter something like "10 k\Omega" in the value field and it could automatically move the units section to the unit field. The only problem with this would be the example Peter gave a while back about `1\Omega5` for `1.5 \Omega`.

**Comment by umesh-timalsina** [comment]
>It might be nice to first implement the first part of that so we can enter something like "10 k\Omega" in the value field and it could automatically move the units section to the unit field. The only problem with this would be the example Peter gave a while back about 1\Omega5 for 1.5 \Omega.

I am not too much worried about the later part as this can be achieved incrementally (e.g. 1\Omega5 or 1.5 \Omega) by having regex support for different use cases. However, if we are to do the first part, I think this should be done separately i.e. #362.

>Good point. We can't really refresh just one of the fields... We could add a refresh button to both. On refresh, we could run OCR then assign the text to the different fields (based on regex from the ontology). Whichever button was refreshed could simply set the field accordingly and the other field (ie, value/unit) could be discarded. 

I think discarding could be an option. For example- Let's say you would do a snippet OCR and get the following value for resistance: `5K`. Ideally, we should be able to split this into two parts 5 for value and `K\Omega` for unit, regardless if the underlying bounding box doesn't change, I think we will end up with the same results. So refreshing `unit` should just change the `unit` field and refreshing `value` should change the `value` field (but results will seem unchanged).
